docs/document/content/user-manual/shardingsphere-scaling/build.cn.md

```diff
@@ -18,7 +18,61 @@ mvn clean install -Dmaven.javadoc.skip=true -Dcheckstyle.skip=true -Drat.skip=tr
 
 或者通过[下载页面]( https://shardingsphere.apache.org/document/current/cn/downloads/ )获取安装包。
 
-2. 解压缩 proxy 发布包，修改配置文件 `conf/server.yaml`，这里主要是开启 `scaling` 和 `mode` 配置：
+> Scaling还是实验性质的功能，在持续进化中，建议使用master分支最新版本，点击此处[下载最新版本]( http://117.48.121.24:8080/Proxy/ )
```

Comment:
It's better not put cn community response here.

Seems the http url ip is hard-coded.

Reply:
Ok，I just want other people know use scaling version is better
and http url ip is your address in README.md

https://github.com/apache/shardingsphere#nightly-builds

So I can use this link replace the old one

Reply:
OK